conf/messages

```diff
@@ -72,31 +72,31 @@ checkBusinessType.progressive.title=My business type is not listed here
 checkBusinessType.progressive.content.p1=You can only get an agent services account if your business is a sole trader, limited company, partnership or limited liability partnership.
```

Comment:
Trivial: checkBusinessType.* messages are now used in the /business-type page rather than /check-business-type.